libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/CustomManifestsForm.tsx

```diff
@@ -36,7 +36,7 @@ const AutosaveWithParentUpdate = ({
   const isEmpty = React.useMemo<boolean>(() => {
     return isEqual(values, emptyValues);
   }, [values, emptyValues]);
-  const isAutoSaveRunning = useFormikAutoSave();
+  const isAutoSaveRunning = useFormikAutoSave(1000);
```

Comment:
`1000` is the default value for debounce. There is no need for this change.

```suggestion
  const isAutoSaveRunning = useFormikAutoSave();
```

Reply:
True. I'll change it